XCUITests/HistoryTests.swift

```diff
@@ -5,6 +5,8 @@
 import XCTest
 
 let webpage = ["url": "www.mozilla.org", "label": "Internet for people, not profit — Mozilla", "value": "mozilla.org"]
+// This is par of the info the user will see in recent closed tabs once the visited website is closed
```

Comment:
par => part

Also, is the reason that "Book of mozilla" is shown in the closed tab list is because we opened it in fxscreengraph during the init?  IF so, we should mention that.

Reply:
We open that site whenever we call ```navigator.goto(BrowserTab)``` which is how the tests start, it is not opened directly in fxscreengraph during the init.

Reply:
Ah right, I know we didn't explicitly mention this in our other XCUITests, but I think this is a good place to start adding a comment saying that.  Readers might wonder just reading from the script "how did this site ended up in our history?"

Reply:
Yeah, PR updated coming with first comment fixed and giving more info: // This is part of the info the user will see in recent closed tabs once the default visited website (https://www.mozilla.org/en-US/book/) is closed
And also before the first ```navigator.goto(BrowserTab)```of each test I have added // Open default website.
Would that be better?